lib/livebook/notebook/explore/elixir_and_livebook.livemd

```diff
@@ -162,50 +140,6 @@ Since this branch is a separate process, a crash has limited scope:
 Process.exit(self(), :kill)
 ```
 
-## Inputs
```

Comment:
Inputs were a Livebook feature, now they are a Kino feature, so I moved this out. We will likely have a new notebook soon, showing how to build a chat app with Kino that will cover inputs, controls, and frames.

Reply:
Yeah, sounds good!